tools/fix_go_deps.sh

```diff
@@ -5,21 +5,27 @@ set -euo pipefail
 : "${GAZELLE_PATH:=}"
 : "${BAZEL_COMMAND:=bazelisk}"
 
-GAZELLE_COMMAND=("$BAZEL_COMMAND" run --build_metadata=DISABLE_COMMIT_STATUS_REPORTING=true //:gazelle --config=buildbuddy_bes_backend --config=buildbuddy_bes_results_url --)
+DIFF_MODE=0
+if [[ "${1:-}" == "-d" ]] || [[ "${1:-}" == "--diff" ]]; then
+  DIFF_MODE=1
+fi
+
+CI_FLAGS=""
+if [[ "${2:-}" == "--ci" ]]; then
+  CI_FLAGS="--config=buildbuddy_bes_backend --config=buildbuddy_bes_results_url"
+fi
+
+
+GAZELLE_COMMAND=("$BAZEL_COMMAND" run --build_metadata=DISABLE_COMMIT_STATUS_REPORTING=true //:gazelle "$CI_FLAGS" --)
 if [[ "$GAZELLE_PATH" ]]; then
   GAZELLE_COMMAND=("$GAZELLE_PATH")
 fi
 
-GO_COMMAND=("$BAZEL_COMMAND" run --build_metadata=DISABLE_COMMIT_STATUS_REPORTING=true //:go --config=buildbuddy_bes_results_url --config=buildbuddy_bes_backend --)
+GO_COMMAND=("$BAZEL_COMMAND" run --build_metadata=DISABLE_COMMIT_STATUS_REPORTING=true //:go "$CI_FLAGS" --)
```

Comment:
`"$CI_FLAGS"` will be expanded to a single variable - if you define CI_FLAGS as an array, like

```
CI_FLAGS=()
if ... then
  CI_FLAGS=(--config=buildbuddy_bes_backend --config=buildbuddy_results_url)
fi
```

then you could use array expansion here:

```suggestion
GO_COMMAND=("$BAZEL_COMMAND" run --build_metadata=DISABLE_COMMIT_STATUS_REPORTING=true "${CI_FLAGS[@]}" //:go --)
```

Reply:
Ah yes - realized this, thanks